ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -2523,6 +2532,7 @@ def setEngine(self,eng,seed=None,count=None):
     if count is None:
       count=self.randomEng.get_rng_state()
     eng.forward_seed(count)
+    print('RNG State:', eng.get_rng_state())
```

Comment:
I think it would be good to either remove the print or use a `self.raiseAMessage()` instead.